web_graph_radar/static/lib/nvd3-radar.js

```diff
@@ -0,0 +1,885 @@
+nv.models.radar = function() {
```

Comment:
Can you add a comment at the top of this file indicating the `nvd3-radar` version that is being used here? Not a blocker, just incredibly helpful from a long-term maintainability perspective IMO